src/platform_impl/linux/window.rs

```diff
@@ -87,6 +89,10 @@ impl Window {
 
     let window = window_builder.build();
 
+    if is_wayland {
+      WlHeader::setup(&window);
```

Comment:
how does this look like on Desktop environments that has server side decorations. I think KDE has server side decorations.

Reply:
In the KDE environment, it looks like this. The button icons are slightly different.
<img width="1002" alt="스크린샷 2024-09-24 오전 3 54 42" src="https://github.com/user-attachments/assets/a3db070f-1819-4189-8b1e-9cce69c6b6c8">

Reply:
apologies, could you show how they compare to another Gtk app? maybe gnome-disks

Reply:
No problem! I ran firefox and gnome-disks. The button styles do look a bit different.
<img width="635" alt="스크린샷 2024-09-25 오전 1 12 30" src="https://github.com/user-attachments/assets/a513ef14-07c3-490c-ba38-558c20c8b84b">
<img width="609" alt="스크린샷 2024-09-25 오전 1 13 44" src="https://github.com/user-attachments/assets/37b6af4d-0da9-46db-bebf-cd6068e5f22a">

Reply:
that's unfortunate, I will look around for a bit and see if we can replicate their behavior.

Reply:
I found a way to align the button styles between non-KDE Wayland and KDE. This works well:

```rust
let header = HeaderBar::builder()
    .show_close_button(true)
    .decoration_layout("menu:minimize,maximize,close")
    .build();
```

ref: https://forum.manjaro.org/t/no-maximize-minimize-buttons-on-flatpak-wayland-gtk-applications/131742

and now, here’s how it looks:

### non-kde wayland
<img width="487" alt="스크린샷 2024-09-25 오후 7 51 39" src="https://github.com/user-attachments/assets/42b7289c-88db-4660-81e1-7503d5cfd875">

#### kde
<img width="739" alt="스크린샷 2024-09-25 오후 7 48 04" src="https://github.com/user-attachments/assets/0828675b-e03e-4fc7-911e-ede8000876c5">

Reply:
Wow!! good job.